packages/rancher-sriov/package.yaml

```diff
@@ -0,0 +1,9 @@
+url: local
```

Comment:
Why are we doing this as a local chart? This chart exists in an upstream https://github.com/k8snetworkplumbingwg/sriov-network-operator/tree/9b59e4ed4785bddeaea67837bd8506163a37a713/deployment/sriov-network-operator.

IMO, we should provide the Github Repository based URL for this chart and maintain just the necessary patches on top of it, unless there is a strong reason why we should not do so.

Reply:
That's a great point. I think the person that started this work missed that there was already a helm chart upstream (possibly the work started before the upstream guys released it). I have just modified my commit and now it is based on the upstream helm chart (following the confluence docs). I tested it and works but please review ;)